<launch>
  <node name="obstacle_detector" pkg="slam" type="obstacle_detector" output="screen"/>
  <node name="pointcloud_data_generator" pkg="slam" type="cluster_to_pointcloud.py" output="screen"/>

  <node name ="obstacle_image" pkg="slam" type="obstacle_image" output="screen"/>

  <node name ="obstacle_costmap_publisher" pkg="slam" type="obstacle_costmap_publisher" output="screen"/>
  <node name ="decaying_costmap_publisher" pkg="slam" type="decaying_costmap_publisher" output="screen"/>
  
  <node name ="parking_spot_detector" pkg="slam" type="parking_spot_detector" output="screen"/>
</launch>
